src/agent/Bradley.kt

```diff
@@ -14,21 +16,18 @@ interface Player {
 }
 
 class GeneralPlayer(override val actor: Actor): Player {
+    private val options = listOf(Vector(1,-1), Vector(1,0), Vector(1, 1),
+    Vector(0,1),Vector(-1, 1), Vector(-1, 0), Vector(-1, -1),
+    Vector(0, -1))
```

Comment:
This is fine, what you were trying to do however is described here:
https://en.wikipedia.org/wiki/Rotation_matrix#In_two_dimensions

and here's one of my implementations of it:
https://github.com/alepar/cubes/blob/master/src/java/main/space3d/Transformations.java#L10

but here, to be honest, simply enumerating all directions is acceptable